pkg/clustermanager/cluster_creator.go

```diff
@@ -0,0 +1,54 @@
+package clustermanager
+
+import (
+	"context"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/filewriter"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/providers"
+	"github.com/aws/eks-anywhere/pkg/types"
+)
+
+type ClusterCreator struct {
+	Applier Applier
+	KM      KubeconfigManager
```

Comment:
in general, i advise to stay away from acronyms unless they refer to well known things
I don't think KM is a well known thing